frontend/src/app/modules/perflow-studio/playlists-page/playlists-page.component.ts

```diff
@@ -1,10 +1,35 @@
 import { Component } from '@angular/core';
+import { filter } from 'rxjs/operators';
+import { PlaylistView } from 'src/app/models/playlist/playlist-view';
+import { AuthService } from 'src/app/services/auth/auth.service';
+import { PlaylistsService } from 'src/app/services/playlists/playlist.service';
 
 @Component({
   selector: 'app-playlists-page',
   templateUrl: './playlists-page.component.html',
   styleUrls: ['./playlists-page.component.sass']
 })
 export class PlaylistsPageComponent {
+  playlists: PlaylistView[] = [];
+  userId: number = 1;
 
+  constructor(
+    private _authService: AuthService,
+    private _playlistService: PlaylistsService
+  ) {
+    this._authService.getAuthStateObservableFirst()
+      .pipe(filter((state) => !!state))
+      .subscribe((authState) => {
+        this.userId = authState!.id;
+      });
+
+    this.loadPlaylist();
+  }
+
+  loadPlaylist() {
+    this._playlistService.getPlaylistsByAuthorId(this.userId)
+      .subscribe((ps) => {
+        this.playlists = ps;
+      });
```

Comment:
You can make it one line without curly braces

Reply:
Lint error: Arrow function should not return assignment  no-return-assign